### PR TITLE
refactor(message): use only one static error component (DEV-900)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DEV-900

### CODE REVIEW

**Comment by kilchenmann** [comment]
@mdelez would it make sense to rename the error-component into status-component, because it's not only used for errors. But we can discuss it on Monday.

**Comment by mdelez** [comment]
> @mdelez would it make sense to rename the error-component into status-component, because it's not only used for errors. But we can discuss it on Monday.

maybe `app-status` would be more clear since "status" can be a bit ambiguous

**Comment by kilchenmann** [comment]
@mdelez the PR should be ready for review now

File: src/app/main/status/status.component.scss [hunk 1]
```diff
@@ -12,24 +12,27 @@
...
+    width: 480px;
+    margin: 0 auto;
 
     .image,
     .text {
       justify-self: stretch;
       align-self: stretch;
       box-sizing: border-box;
-      padding: 24px;
+      // padding: 24px;
```
Comment (mdelez) [inline src/app/main/status/status.component.scss:23]: can this be removed?

File: src/app/user/user-form/user-form.component.ts [hunk 2]
```diff
@@ -384,33 +381,35 @@ export class UserFormComponent implements OnInit, OnChanges {
...
+                        session.user.lang = this.userForm.controls['lang'].value;
                         localStorage.setItem('session', JSON.stringify(session));
                     }
 
                     this._cache.set(this.username, response);
 
-                    this.success = true;
-
+                    // this.loading = false;
+                    this._notification.openSnackBar('You have successfully updated user\'s profile data.');
```
Comment (mdelez) [inline src/app/user/user-form/user-form.component.ts:391]: **the** user's profile data

**Review by mdelez — APPROVED** [review]
Thank you for this! A few remarks but then it can be merged.

File: src/app/user/user-form/user-form.component.ts (hunk 2 above)
Comment (kilchenmann) [reply src/app/user/user-form/user-form.component.ts]: resolved in a3fdd232

**Pull request merged**